getMongoData/getMongoData.js

```diff
@@ -302,6 +302,27 @@ function printDataInfo(isMongoS) {
                     }
                     printInfo('Indexes',
                               function(){return db.getSiblingDB(mydb.name).getCollection(col).getIndexes()}, section);
+                    printInfo('Index Stats',
+                              function(){var res = db.getSiblingDB(mydb.name).runCommand( {
+                                aggregate: col,
+                                pipeline: [
+                                  {$indexStats: {}},
+                                  {$group: {_id: "$key", stats: {$push: {accesses: "$accesses.ops", host: "$host", since: "$accesses.since"}}}},
```

Comment:
Can you explain the need for this?  Presumably it's for when running against a sharded cluster via mongos, and obtaining results from several shards?